src/pykeen/models/base.py

```diff
@@ -803,6 +673,22 @@ def score_r(self, ht_batch: torch.LongTensor) -> torch.FloatTensor:
         scores = expanded_scores.view(ht_batch.shape[0], -1)
         return scores
 
+    def compute_loss(
+        self,
+        tensor_1: torch.FloatTensor,
+        tensor_2: torch.FloatTensor,
+    ) -> torch.FloatTensor:
+        """Compute the loss for functions requiring two separate tensors as input.
+
+        :param tensor_1: shape: s
+            The tensor containing predictions or positive scores.
+        :param tensor_2: shape: s
```

Comment:
generally the two tensors do not need to have the same shape, but only one which is broadcastable

Reply:
oh good point, I copied this and didn't update it. Would you please re-write this description to be more accurate?

Reply:
done in e6ce485a